ethereum.md

```diff
@@ -86,7 +96,7 @@ To do so, we'll extend sort `JSON` with some IELE specific syntax, and provide a
 -   `#adjustGas` fakes the gas usage of the transaction since the EVM-to-IELE conversion does not preserve gas usage.
 -   `finishTx` is a place-holder for performing necessary cleanup after a transaction.
 
-```{.k .uiuck .rvk}
+```{.k .uiuck .rvk .standalone .node}
```

Comment:
If I understand it correctly, the entry point for the .node version should be `runVM`, why include startTx, loadTx, and etc in the .node version?

Reply:
I have disabled more of the code in node that is not needed.